src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs

```diff
@@ -635,8 +635,6 @@ protected override void OnKeyDown(KeyRoutedEventArgs args)
 			{
 				case VirtualKey.Left:
 				case VirtualKey.Right:
-				case VirtualKey.Up:
-				case VirtualKey.Down:
```

Comment:
I wonder if these should be set as handled in the case of a TextBox that allows multiline input (ie `AcceptsReturn="True"`). Not necessarily a big deal in the context of the current PR, though.

Reply:
That's a good idea actually. I could theoretically just put the Up/Down cases first and break if acceptsreturn is false.